.github/workflows/shipping-rate-presenter.rust.default copy.yml

```diff
@@ -0,0 +1,30 @@
+name: Rust
```

Comment:
Is this file named as "xxx.default copy.yml"?

Reply:
Oops, forgot to move the `copy`